components/TextArea/TextArea.jsx

```diff
@@ -13,10 +13,12 @@ import {
 const ID_GENERATOR = uniqId('textarea-');
 
 const TextAreaTag = styled(TextInput)`
-  display: block;
-  min-height: 108px;
-  margin-top: 8px;
-  transition: border 0.2s ease-in-out, box-shadow 0.2s ease-in-out;
+  && {
+    display: block;
+    min-height: 108px;
+    margin-top: 8px;
```

Comment:
here we should use the spacing from theme.
use the `Input.jsx` as reference.
Do not forget of using the same props from Input, because the TextInput constant.

Ps: updates the margin-top in same line, that way the snapshots will not be updated.
![image](https://user-images.githubusercontent.com/3389749/130670902-5e05a2a3-510c-44f8-b028-6f2d295d8baa.png)